docs/readthedocs/about.md

```diff
@@ -1,10 +1,20 @@
 (about)=
-# About SIMONA
+# About Simona
 
-## Overview
+Simona is a simulation-tool for generating time-series in the context of distribution grids to be used in grid planning, operation and analysis. The simulation is executed using an agent-based, discrete-event model. Simona is being developed at the [Institute of Energy Systems, Energy Efficiency and Energy Economics](https://ie3.etit.tu-dortmund.de) at [TU Dortmund University](https://www.tu-dortmund.de), Germany.
 
-SIMONA is an agent-based discrete-event power system simulation model developed at the [Institute of Energy Systems, Energy Efficiency and Energy Economics](https://ie3.etit.tu-dortmund.de) at [TU Dortmund University](https://www.tu-dortmund.de), Germany.
+## Power Grid Model
+With Simona, you can model your power grid using a broad variety of different available asset types. The input data for the grid must be provided as PSDM-data. (For more information on this data type, go to the [PSDM](https://powersystemdatamodel.readthedocs.io/en/latest/index.html) documentation.) The combination of implemented physical participant models and your provided input data forms a power grid that can be further investigated and analysed using Simona services.
```

Comment:
I would suggest to have the Hyperlinks also with a floating window (check the link to PSDM at Line Model for example https://simona--199.org.readthedocs.build/en/199/models/line_model.html)